cluster/gce/configure-vm.sh

```diff
@@ -82,6 +73,19 @@ for k,v in yaml.load(sys.stdin).iteritems():
   fi
 }
 
+function maybe-get-kube-token() {
```

Comment:
I think `ensure-` is the convention for these semantics? I think it's also a little clearer.